generic/tests/vlan.py

```diff
@@ -134,6 +137,20 @@ def flood_ping(src, dst):
                         session=session_flood, timeout=10)
         session_flood.close()
 
+    def bridge_vlan_config():
+        """
+        Configure vlan over the bridge interface
+        """
+        ip_cmd = "ip link"
+        ifconfig_cmd = "ifconfig"
```

Comment:
```python
ip_cmd = "ip link"
ifconfig_cmd = "ifconfig"
```
you can put these two lines directly in your cmds, you don't need to format it into your cmd.